Templates/Ros2ProjectTemplate/Template/Levels/DemoLevel/DemoLevel.prefab

```diff
@@ -167,6 +168,187 @@
                 }
             }
```

Comment:
It works really great. I have just one suggestion: I'd add marker points (in addition to labels). Something like in the image below. Suggested solution in next comments.

![Screenshot from 2023-05-12 21-20-10](https://github.com/o3de/o3de-extras/assets/31925544/7149d48e-1100-47aa-bf08-00146ff618c2)

Reply:
Agreed